apps/test/unit/blockly/customFunctionsTest.js

```diff
@@ -1,52 +0,0 @@
-import {expect} from '../../util/reconfiguredChai';
```

Comment:
Just curious if there is discussion/plan for how to test new function editor.

Reply:
not yet, it might be best tested as a UI test? In general I need to figure out how we want to test blockly code.